File: VB/CustomMVCHelpers/Controllers/HomeController.vb
Imports Microsoft.VisualBasic
Imports MVCxGridViewDataBinding.Models
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web.Mvc

Namespace CustomMVCHelpers.Controllers
	Public Class HomeController
		Inherits Controller
		Public Function Index() As ActionResult
			Return View()
		End Function
		Public Function GridViewPartial() As ActionResult
			Dim model = MyModel.GetModelList()
			Return PartialView(model)
        End Function
	End Class
End Namespace

File: CS/CustomMVCHelpers/Controllers/HomeController.cs
using MVCxGridViewDataBinding.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web.Mvc;

namespace CustomMVCHelpers.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            return View();
        }
        public ActionResult GridViewPartial()
        {
            var model = MyModel.GetModelList();
            return PartialView(model);
        }
    }
}
